skills/freee-api-skill/references/hr-groups.md

```
@@ -0,0 +1,52 @@
+**操作**: 所属一覧の取得
+
+**説明**: 概要 指定した事業所の指定日付時点における所属情報をリストで返します。 注意点 管理者権限を持ったユーザーのみ実行可能です。
+
+### パラメータ
+
+| 名前 | 位置 | 必須 | 型 | 説明 |
+|------|------|------|-----|------|
+| company_id | query | はい | integer | 事業所ID |
+| base_date | query | はい | string(date) | 指定日。指定日付時点における所属情報をリストで返します。(YYYY-MM-DD)(例:2018-07-31) |
+| with_no_payroll_calculation | query | いいえ | boolean | trueを指定すると給与計算対象外の従業員情報をレスポンスに含めます。 |
+| employee_ids | query | いいえ | string | 取得対象とする従業員IDを指定することができます。指定しない場合は全従業員が対象となります。
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)